<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="TextFreq_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeekDown_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TuneDown_Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TuneUp_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeekUp_Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Frequency_LCD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="SeekDown_Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seek down</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TuneDown_Label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tune down</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TuneUp_Label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tune up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SeekUp_Label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seek up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RDS_Ind_Label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RDS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TP_Ind_Label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Signal_dBuV_LCD">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Signal_Label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Stereo_Ind_Label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="RDSRT_TextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Radiotext_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radiotext:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RDSPI_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RDS PI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextFreq_label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RDS PS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RDS_PI_QLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>----</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RDS_PS_QLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>--------</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="SeekSensitivity_Combo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SeekSensitivity_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seek sensitivity</string>
    </property>
   </widget>
   <widget class="QLabel" name="TuneSteps_Label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tune steps</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TuneSteps_Combo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>83</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTEF_6686_Tuner">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_to_tuner"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuTEF_6686_Tuner"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConnect_to_tuner">
   <property name="text">
    <string>Connect to tuner</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
